src/main/java/seedu/address/logic/commands/DoneCommand.java

```diff
@@ -19,19 +22,26 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Marks a task as COMPLETED.\n"
```

Comment:
Would be good to say the range of the rating (i.e. 1-5) here

Reply:
done